bedrock/products/redirects.py

```diff
@@ -5,6 +5,8 @@
 from bedrock.redirects.util import redirect
 
 redirectpatterns = (
+    # Issue 10335
+    redirect(r"^vpn/?$", "products.vpn.landing"),
```

Comment:
Suggestion: worth adding a test for this existing redirect?

Reply:
It already exists here from a previous time: https://github.com/mozilla/bedrock/blob/main/tests/redirects/map_globalconf.py#L1176
I added it in this PR some time ago: https://github.com/mozilla/bedrock/pull/10338/files#diff-f0a4473a733687326b397105e2d566e6c377bf59fc212f3032e6e4a2882d6551R1393